README.md

```diff
@@ -10,5 +10,7 @@ Dimensions is not part of the Sugar desktop, but can be added.  Please refer to;
 
 * [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
 * [How to use Sugar](https://help.sugarlabs.org/),
-* [Download Dimensions using Browse](https://activities.sugarlabs.org/), search for `Dimensions`, then download, and;
+* [Download Dimensions using Browse for sugar 0.116 or lower](https://activities.sugarlabs.org/),
+* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
+* search for `Dimensions`, then download, and;
```

Comment:
Both these, should be on the same item.

Reply:
Can you explain a bit?  What do you mean by under the same item ??

Reply:
```diff
+* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
+ search for `Dimensions`, then download, and;
```

like this...

Reply:
[Download Dimensions using Browse for sugar 0.116 or lower](https://activities.sugarlabs.org/),
* search for `Dimensions`, then download, and;
* [Download Dimensions using Browse for sugar 0.117 or greater](https://v4.activities.sugarlabs.org/)
* search for `Dimensions`, then download, and;

Like this ??